package.json

```diff
@@ -34,6 +33,7 @@
     "dockerode": "^2.5.0",
     "fs-extra": "^4.0.2",
     "jsrsasign": "^8.0.4",
+    "kafka-node": "^2.3.0",
```

Comment:
kafka-node is not a necessary package for caliper users, I think it should be removed.